### DISCUSSION TITLE
Merge `@luciferreeves/blaver` to this repo

### DISCUSSION THREAD

**Discussion by luciferreeves** [faker-js/faker#165] [General]
Hello,

A few days ago, I received an issue on [`@luciferreeves/blaver`](https://github.com/luciferreeves/blaver) to merge the project with this repo and maintain this project instead. Now, I'm not going to try and read the tea leaves and figure out what the future looks like. The question for me was, "Do I maintain the repo alone or move to a more community based approach?" I thought the answer was obvious and that I can maintain `blaver`, but now I'm convinced of the opposite. 

I pondered over the question for a few days and I thought, its much easier to contribute to a public repository rather than a repo that I handle myself. I would continue to improve faker.js but I think it's much better if I do this here. 

So, I would like to retire/archive the repo and update the readme of the project once I have a confirmation here and I would deprecate the [package on NPM](https://www.npmjs.com/package/blaver), and link to release of [`faker-js/faker`](https://github.com/faker-js/faker). 

I also registered a domain ([blaver.dev](https://blaver.dev)), which I can point to [fakerjs.dev](https://fakerjs.dev/), if you guys want that. Let me know what you think.

**Comment by Shinigami92** [comment]
You are absolutely welcome 🤗 
Please show up in our Discord server and tell us there your name, than we can proceed further.

**Comment by damienwebdev** [comment]
@luciferreeves welcome our not-so-little little community! As @Shinigami92 mentioned, if you hop into discord (don't forget to give us your name here so that we know who you are) we can add roles!

Additionally, I'm going to move this to a discussion since it is not a code related issue.

Further, you may note our repo has a few new processes beyond the original Faker.js that your fork was based upon. If you have any questions, please feel free to ask!

Finally, if you have any cool ideas that your learned from working on your fork, we'd welcome any contribution either in idea or in code!

**Comment by ejcheng** [comment]
Welcome! Just curious, in the week or so since the original repo was destroyed, have you made any enhancements on top of the old Faker? We would love to have that code if we decide to merge :)

**Comment by fzn0x** [comment]
Your welcome seriously, let's maintain and make this community bigger, as an alternative of Marak faker.js

**Comment by luciferreeves** [comment]
Actually, I was building a static website for blaver, If you guys like it, I can merge the code. I also thought of adding random Zodiac signs too. Probably move to a tree-shakeable layout for the library, but that seems a long shot. Also, I used pravatar instead of fakercloud. Those are the only changes till now, apart from some refactoring changes.

**Reply by ejcheng** [reply]
@Shinigami92 @damienwebdev We could add random zodiac signs for v6.0.0, or should we wait to implement that until 6.1.0?

I'm not sure about the other stuff.

**Reply by Shinigami92** [reply]
You should create an issue for such thinks :wink:

**Reply by ejcheng** [reply]
> You should create an issue for such thinks 😉

We should let @luciferreeves do that, he should get credit for his idea haha

**Reply by luciferreeves** [reply]
lol... @import-brain. I will do that. No Problem. (●´艸` )
